metatag.module

```diff
@@ -2137,11 +2143,6 @@ function metatag_preprocess_page(&$variables) {
  * @see metatag_preprocess_page().
  */
 function metatag_preprocess_maintenance_page(&$variables) {
-  // Don't modify install or update pages.
-  if (defined('MAINTENANCE_MODE') && in_array(MAINTENANCE_MODE, array('install', 'update'))) {
-    return;
-  }
-
```

Comment:
This change here seems to be unrelated to the problem we are trying to fix.

Reply:
No idea how I managed to delete this.